modular_skyrat/modules/mapping/code/mob_spawns.dm

```diff
@@ -341,6 +348,15 @@
 	back = /obj/item/storage/backpack
 	id = /obj/item/card/id/away/freightcrew
 
+/datum/outfit/freighter_crew/post_equip(mob/living/carbon/human/crewman, visualsOnly)
+	var/obj/item/card/id/id_card = crewman.wear_id
+	if(istype(id_card))
+		id_card.registered_name = crewman.real_name
+		id_card.update_label()
+		id_card.update_icon()
+	handlebank(crewman)
```

Comment:
For all of these, please just put a space before the `handlebank()` call please <3
```suggestion

	handlebank(crewman)
```